api/api.go

```diff
@@ -185,17 +216,35 @@ func new(cfg *config.TemporalConfig, router *gin.Engine, ipfs rtfs.Manager, debu
 		signer:  signer,
 		orch:    orch,
 		dc:      dc,
-		zm:      models.NewZoneManager(dbm.DB),
-		rm:      models.NewRecordManager(dbm.DB),
-		nm:      models.NewHostedIPFSNetworkManager(dbm.DB),
+		queues: queues{
+			pin:      qmPin,
+			file:     qmFile,
+			cluster:  qmCluster,
+			email:    qmEmail,
+			ipns:     qmIpns,
+			key:      qmKey,
+			database: qmDatabase,
+		},
+		zm: models.NewZoneManager(dbm.DB),
+		rm: models.NewRecordManager(dbm.DB),
+		nm: models.NewHostedIPFSNetworkManager(dbm.DB),
 	}, nil
 }
 
 // Close releases API resources
 func (api *API) Close() {
+	// close grpc connections
 	api.lc.Close()
 	api.signer.Close()
 	api.orch.Close()
+	// close queue resources
+	api.queues.cluster.Close()
+	api.queues.database.Close()
+	api.queues.email.Close()
+	api.queues.file.Close()
+	api.queues.ipns.Close()
+	api.queues.key.Close()
+	api.queues.pin.Close()
```

Comment:
weeee probs no way to do this better but damn thats a lotta boilerplate and things we might forget to `Close()` lmao

Reply:
Haha yea can't think of a better way :frowning_face:  I added some logging so we can tell after the fact whether or not the channels were properly closed